src/pages/confs/2017_jsconf_eu/immutable_data_structures_for_functional_js.md

```diff
@@ -55,4 +55,11 @@ Related Links | Rating
 [mori](http://swannodette.github.io/mori/) | ★★★☆☆
 [immutable-js](https://facebook.github.io/immutable-js/) | ★★★☆☆
 
-</article>
\ No newline at end of file
+</article>
+
+<article id="4">
+
+[Video on Youtube](https://www.youtube.com/watch?v=Wo0qiGPSV-s)
```

Comment:
It's better not to create specific markup for specific talks. Otherwise, things become very inconsistent very fast. Youtube videos are added via the `video` metadata field. This one is already added in master as well.